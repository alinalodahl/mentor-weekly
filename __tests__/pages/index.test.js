/* eslint-env jest */

import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import App from "../../pages/index.js";

describe("With Enzyme", () => {
  it("Renders without crashing", () => {
    shallow(<App />);
  });
  it('App shows "Mentor Weekly"', () => {
    const app = shallow(<App />);

    expect(app.find("h1")).toEqual("Mentor Weekly");
  });
});

// describe("With Snapshot Testing", () => {
//   it('App shows "Hello world!"', () => {
//     const component = renderer.create(<App />);
//     const tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });
