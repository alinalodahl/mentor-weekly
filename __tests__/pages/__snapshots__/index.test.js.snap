// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div
  className="jsx-732902753 jsx-732902753"
>
  <div
    className="jsx-732902753 jsx-732902753 wrap"
  >
    <div
      className="jsx-2059778970 jsx-2059778970 nav landing-nav"
    >
      <div
        className="jsx-2059778970 jsx-2059778970 logo"
      >
        <img
          alt="wocintech stock photo"
          className="jsx-2059778970 jsx-2059778970 mw-logo"
          src="../static/mentor-weekly-logo.svg"
          width="100%"
        />
      </div>
      <div
        className="jsx-2059778970 jsx-2059778970 right landing-right"
      >
        <span
          className="jsx-3921880395 jsx-3921880395 button-span"
        >
          <div
            className="jsx-3921880395 jsx-3921880395 button-div"
          >
            <button
              className="jsx-3921880395 jsx-3921880395 button small"
              onClick={[Function]}
              type="button"
            >
              log in
            </button>
          </div>
        </span>
        <span
          className="jsx-908513797 jsx-908513797 button-span"
        >
          <div
            className="jsx-908513797 jsx-908513797 button-div"
          >
            <a
              className="jsx-908513797 jsx-908513797"
              href="#signup"
              onClick={[Function]}
            >
              <button
                className="jsx-908513797 jsx-908513797 button small"
                onClick={null}
                type="button"
              >
                sign up
              </button>
            </a>
          </div>
        </span>
      </div>
    </div>
    <div
      className="jsx-732902753 jsx-732902753 landing-page"
    >
      <div
        className="jsx-732902753 jsx-732902753 landing-image-div"
      >
        
        <div
          className="jsx-732902753 jsx-732902753 title-div"
        >
          <h1
            className="jsx-732902753 jsx-732902753 title"
          >
            mentor weekly
          </h1>
          <h2
            className="jsx-732902753 jsx-732902753 tagline"
          >
            a better mentoring platform for those in tech
          </h2>
        </div>
      </div>
      <div
        className="jsx-732902753 jsx-732902753 entry-point-button-area"
      >
        <span
          className="jsx-908513797 jsx-908513797 button-span"
        >
          <div
            className="jsx-908513797 jsx-908513797 button-div"
          >
            <a
              className="jsx-908513797 jsx-908513797"
              href="/#signup"
              onClick={[Function]}
            >
              <button
                className="jsx-908513797 jsx-908513797 button large"
                onClick={null}
                type="button"
              >
                get involved
              </button>
            </a>
          </div>
        </span>
        <span
          className="jsx-2155738629 jsx-2155738629 button-span"
        >
          <div
            className="jsx-2155738629 jsx-2155738629 button-div"
          >
            <a
              className="jsx-2155738629 jsx-2155738629"
              href="#infographic"
              onClick={[Function]}
            >
              <button
                className="jsx-2155738629 jsx-2155738629 button large"
                onClick={null}
                type="button"
              >
                learn more
              </button>
            </a>
          </div>
        </span>
      </div>
      <div
        className="jsx-732902753 jsx-732902753"
      >
        <div
          className="jsx-3219844834 jsx-3219844834 infographic"
          id="infographic"
        >
          <div
            className="jsx-3219844834 jsx-3219844834 image-div"
          />
          <div
            className="jsx-3219844834 jsx-3219844834 text-div"
          >
            <div
              className="jsx-3219844834 jsx-3219844834 text-inner-div"
            >
              <h3
                className="jsx-3219844834 jsx-3219844834 header"
              >
                gain candidates
              </h3>
              <p
                className="jsx-3219844834 jsx-3219844834 text"
              >
                As the tech industry continues to grow and innovate, employers need a way to access quality candidates. Mentor Weekly provides employers with a potential hiring pool of professionals that their employees already have experience with. Mentors gain leadership skills, and companies invest in the future of the field. 
              </p>
            </div>
          </div>
        </div>
        <div
          className="jsx-160653531 jsx-160653531 infographic"
          id="infographic"
        >
          <div
            className="jsx-160653531 jsx-160653531 image-div"
          />
          <div
            className="jsx-160653531 jsx-160653531 text-div"
          >
            <div
              className="jsx-160653531 jsx-160653531 text-inner-div"
            >
              <h3
                className="jsx-160653531 jsx-160653531 header"
              >
                gain connections
              </h3>
              <p
                className="jsx-160653531 jsx-160653531 text"
              >
                Professionals new to tech need experience, advice, and most importantly, networking opportunities. Mentoring gives junior engineers valuable insights not only into coding, but into the roles, tools, and strategies that are essential to their chosen field.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-4064721996 jsx-4064721996 largequote-div"
      >
        <div
          className="jsx-4064721996 jsx-4064721996 largequote"
        >
          Mentor Weekly
          <span
            className="jsx-4064721996 jsx-4064721996 orange"
          >
             matches mentors & mentees 
          </span>
          based on their strengths and goals.
        </div>
      </div>
      <div
        className="jsx-732902753 jsx-732902753 screenshot-div"
      >
        <img
          className="jsx-732902753 jsx-732902753 screenshot"
          src="/static/dashboard-screen-shot.jpg"
        />
      </div>
      <div
        className="jsx-732902753 jsx-732902753 bubblequotes-div"
      >
        <div
          className="jsx-2717389610 jsx-2717389610 bubblequote-div"
        >
          <div
            className="jsx-2717389610 jsx-2717389610 bubblequote"
          >
            <div
              className="jsx-2717389610 jsx-2717389610 text"
            >
              “Feeling a little uncomfortable with your skills is a sign of learning, and continuous learning is what the tech industry thrives on! It’s important to seek out environments where you are supported, but where you have the chance to be uncomfortable and learn new things.”
            </div>
          </div>
          <div
            className="jsx-2717389610 jsx-2717389610 cite-div"
          >
            <div
              className="jsx-2717389610 jsx-2717389610 speech-bubble"
            >
              <img
                className="jsx-2717389610 jsx-2717389610"
                src="/static/speech-bubble.svg"
              />
            </div>
            <div
              className="jsx-2717389610 jsx-2717389610 author-div"
            >
              Vanessa Hurst
            </div>
            Co-founder of 
            <em
              className="jsx-2717389610 jsx-2717389610"
            >
              Girl Develop It
            </em>
          </div>
        </div>
        <div
          className="jsx-1602283676 jsx-1602283676 bubblequote-div"
        >
          <div
            className="jsx-1602283676 jsx-1602283676 bubblequote"
          >
            <div
              className="jsx-1602283676 jsx-1602283676 text"
            >
              “If you want to grow as a Software Engineer, teach what you’ve learned to others as soon as possible.”
            </div>
          </div>
          <div
            className="jsx-1602283676 jsx-1602283676 cite-div"
          >
            <div
              className="jsx-1602283676 jsx-1602283676 speech-bubble"
            >
              <img
                className="jsx-1602283676 jsx-1602283676"
                src="/static/speech-bubble.svg"
              />
            </div>
            <div
              className="jsx-1602283676 jsx-1602283676 author-div"
            >
              Ellie Day
            </div>
            Founder of 
            <em
              className="jsx-1602283676 jsx-1602283676"
            >
              Mentor Weekly
            </em>
          </div>
        </div>
        <div
          className="jsx-2717389610 jsx-2717389610 bubblequote-div"
        >
          <div
            className="jsx-2717389610 jsx-2717389610 bubblequote"
          >
            <div
              className="jsx-2717389610 jsx-2717389610 text"
            >
              “Without having mentors at an authoritative level saying, ‘You deserve this sort of experience; you should apply for this program, because you’re strong enough‘; those are the sorts of things I wouldn’t necessarily have done without a strong mentoring voice to push me.”
            </div>
          </div>
          <div
            className="jsx-2717389610 jsx-2717389610 cite-div"
          >
            <div
              className="jsx-2717389610 jsx-2717389610 speech-bubble"
            >
              <img
                className="jsx-2717389610 jsx-2717389610"
                src="/static/speech-bubble.svg"
              />
            </div>
            <div
              className="jsx-2717389610 jsx-2717389610 author-div"
            >
              Emily Whiting
            </div>
            Asst. Professor of Computer Science, 
            <em
              className="jsx-2717389610 jsx-2717389610"
            >
              Dartmouth
            </em>
          </div>
        </div>
      </div>
      <div
        className="jsx-4064721996 jsx-4064721996 largequote-div"
      >
        <div
          className="jsx-4064721996 jsx-4064721996 largequote"
        >
          When mentors invest
          <span
            className="jsx-4064721996 jsx-4064721996 orange"
          >
             30 minutes a week 
          </span>
          advising a professional just starting in tech, employers, mentors, and mentees win.
        </div>
      </div>
      <div
        className="jsx-1261211817 jsx-1261211817 signup"
        id="signup"
      >
        <h2
          className="jsx-1261211817 jsx-1261211817"
        >
          get involved
        </h2>
        <form
          className="jsx-1261211817 jsx-1261211817 signup-form"
        >
          <div
            className="jsx-1261211817 jsx-1261211817 radio-div"
          >
            <legend
              className="jsx-1261211817 jsx-1261211817"
            >
              Which role are you interested in?
            </legend>
            <input
              className="jsx-1261211817 jsx-1261211817 radio-button"
              id="signup"
              name="signup"
              onClick={[Function]}
              type="radio"
              value="mentor"
            />
            <label
              className="jsx-1261211817 jsx-1261211817"
              htmlFor="signup"
            >
              being a mentor
            </label>
            <input
              className="jsx-1261211817 jsx-1261211817 radio-button"
              id="signup"
              name="signup"
              onClick={[Function]}
              type="radio"
              value="mentee"
            />
            <label
              className="jsx-1261211817 jsx-1261211817"
              htmlFor="signup"
            >
              being a mentee
            </label>
          </div>
          <div
            className="jsx-3110373596 jsx-3110373596 mentor-div"
          >
            <label
              className="jsx-3110373596 jsx-3110373596 name-label first-name"
              htmlFor="firstName"
            >
              first name
            </label>
            <input
              className="jsx-3110373596 jsx-3110373596 narrow-input"
              id="firstName"
              onChange={[Function]}
              placeholder="first name"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 name-label"
              htmlFor="lastName"
            >
              last name
            </label>
            <input
              className="jsx-3110373596 jsx-3110373596 narrow-input"
              id="lastName"
              onChange={[Function]}
              placeholder="last name"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="contact"
            >
              what email address can we use to contact you about a mentorship?
              (when you create a password, we will send a verification code to this address)
            </label>
            <input
              className="jsx-3110373596 jsx-3110373596 narrow-input"
              id="contact"
              onChange={[Function]}
              placeholder="ex: someone@yahoo.com"
              type="contact"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="goals"
            >
              please describe your goals and what you'd like to gain from a mentorship:
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="goals"
              onChange={[Function]}
              placeholder="ex: meet aspiring front-end engineers"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="organization"
            >
              which organization do you currently work for? What is the main mission/product?
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="organization"
              onChange={[Function]}
              placeholder="ex: freelance, yelp"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="experience"
            >
              how many years of experience do you have?
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="experience"
              onChange={[Function]}
              placeholder="ex: 8 years as a QA engineer"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="skills"
            >
              what areas of expertise could you share with a mentee?
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="skills"
              onChange={[Function]}
              placeholder="ex: career advice, JavaScript, NodeJS, GIT, SQL, NoSQL, React"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="looking-for"
            >
              what are you looking for in a mentee?
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="looking-for"
              onChange={[Function]}
              placeholder="ex: interest in QA testing"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="portfolioUrl"
            >
              please enter the url of your portfolio or website:
            </label>
            <input
              className="jsx-3110373596 jsx-3110373596 narrow-input"
              id="portfolioUrl"
              onChange={[Function]}
              placeholder="ex: github.com/myrepo"
              type="text"
              value=""
            />
            <label
              className="jsx-3110373596 jsx-3110373596 block"
              htmlFor="photoUrl"
            >
              paste the url of the photo you want as your profile picture here (you can add this later):
            </label>
            <textarea
              className="jsx-3110373596 jsx-3110373596 block block-input"
              id="photoUrl"
              onChange={[Function]}
              placeholder="ex: https://instagram.fsnc1-1.fna.fbcdn.net/n680_n.jpg"
              type="text"
              value=""
            />
            <span
              className="jsx-2232346917 jsx-2232346917 button-span"
            >
              <div
                className="jsx-2232346917 jsx-2232346917 button-div"
              >
                <button
                  className="jsx-2232346917 jsx-2232346917 button medium"
                  onClick={[Function]}
                  type="submit"
                >
                  join mentor weekly
                </button>
              </div>
            </span>
            
          </div>
        </form>
      </div>
      
    </div>
  </div>
  <div
    className="jsx-1335444172 jsx-1335444172"
  >
    <div
      className="jsx-1335444172 jsx-1335444172 footer-div"
    >
      <div
        className="jsx-1335444172 jsx-1335444172 logo"
      >
        <img
          alt="wocintech stock photo"
          className="jsx-1335444172 jsx-1335444172 mw-logo"
          src="../static/mw-logo.svg"
          width="100%"
        />
      </div>
      <div
        className="jsx-1335444172 jsx-1335444172 buttons"
      >
        <span
          className="jsx-753555548 jsx-753555548 button-span"
        >
          <div
            className="jsx-753555548 jsx-753555548 button-div"
          >
            <button
              className="jsx-753555548 jsx-753555548 button small"
              onClick={null}
              type="button"
            >
              	©2018 Mentor Weekly
            </button>
          </div>
        </span>
        <p
          className="jsx-1335444172 jsx-1335444172 bullet"
        >
          ·
        </p>
        <span
          className="jsx-753555548 jsx-753555548 button-span"
        >
          <div
            className="jsx-753555548 jsx-753555548 button-div"
          >
            <button
              className="jsx-753555548 jsx-753555548 button small"
              onClick={[Function]}
              type="button"
            >
              Photo Credits
            </button>
          </div>
        </span>
      </div>
    </div>
  </div>
</div>
`;
